lib/licensee/commands/detect.rb

```diff
@@ -13,6 +13,7 @@ class LicenseeCLI < Thor
   option :confidence, type: :numeric, default: Licensee.confidence_threshold, desc: 'Confidence threshold'
   option :license, type: :string, desc: 'The SPDX ID or key of the license to compare (implies --diff)'
   option :diff, type: :boolean, desc: 'Compare the license to the closest match'
+  option :ref, type: :string, desc: 'The name of the commit/branch/tag to search (github.com PATHs only)'
```

Comment:
`PATHs` is somewhat confusing terminology, wonder if this should just be:
```suggestion
  option :ref, type: :string, desc: 'The name of the commit/branch/tag to search (github.com only)'
```

I might be missing something but a chatbot seems to agree 🙃 

> Certainly! Let's dive into the world of Git and GitHub paths.
> 
> 1. **Git Reference (Git Ref)**:
>     - A **Git reference** (often called a **git ref**) is a file that contains a **Git commit SHA-1 hash**.
>     - When referring to a specific Git commit, you can use the **Git reference** as an easy-to-remember name instead of dealing with the raw hash.
>     - Git references can be rewritten to point to new commits. For example:
>         - A **branch** is a Git reference that stores the latest Git commit hash for that branch.
>         - A **tag** is another type of Git reference that points to a specific commit (usually used for releases).
>     - Instead of saying "commit `a3b7f9c2e`," you can say "the `main` branch" or "the `v1.0` tag."
> 
> 2. **GitHub PATH**:
>     - The term "GitHub PATH" might be a bit confusing because it's not a standard Git concept.
>     - However, I suspect you might be referring to **GitHub REST API endpoints** related to Git references.
>     - These endpoints allow you to interact with references (branches, tags, etc.) in your Git database on GitHub.
>     - For example, you can use the GitHub API to:
>         - List matching references based on a name (e.g., branches or tags).
>         - Retrieve information about a specific reference (e.g., commit SHA-1 hash).
>         - Create or update references.
>     - The **GitHub PATH** here refers to the URL path you use to access these API endpoints.
> 
> 3. **Example API Endpoint**:
>     - To list matching references (branches or tags) in a GitHub repository, you can use the following API endpoint:
>         - `GET /repos/{owner}/{repo}/git/matching-refs/{ref}`
>     - Here:
>         - `{owner}` is the account owner (case-insensitive).
>         - `{repo}` is the repository name (without the `.git` extension).
>         - `{ref}` can be a commit SHA, branch name (e.g., `heads/main`), or tag name (e.g., `tags/v1.0`).
>     - The response will include information about the matching references.
> 
> 4. **Example Usage**:
>     - Suppose you want to list matching references for the `main` branch in a repository owned by `octocat`. You'd make a request like this:
>         - `GET /repos/octocat/my-repo/git/matching-refs/heads/main`
> 
> Remember, Git references and GitHub API endpoints play a crucial role in managing version control and collaboration within repositories! [🌟](https://docs.github.com/en/rest/git/refs)¹²³..
> 
> Source: Conversation with Bing, 3/[1](https://docs.github.com/en/rest/git/refs)[2](https://git-scm.com/book/en/v2/Git-Internals-The-Refspec)/2024
> (1) REST API endpoints for Git references - GitHub Docs. https://docs.github.com/en/rest/git/refs.
> (2) Git - The Refspec. https://git-scm.com/book/en/v2/Git-Internals-The-Refspec.
> ([3](https://superuser.com/questions/614978/how-to-reference-a-git-repository)) path - How to reference a git repository? - Super User. https://superuser.com/questions/614978/how-to-reference-a-git-repository.
> (4) Git - git-show-ref Documentation. https://git-scm.com/docs/git-show-ref.
> (5) undefined. https://api.github.com/repos/OWNER/REPO/git/matching-refs/REF.
> (6) undefined. https://github.com/schacon/simplegit-progit.

(First response related to file and executable paths, which I guess will be what most people associate path with.)

Reply:
Alternately, 
```suggestion
  option :ref, type: :string, desc: 'The name of the commit/branch/tag to search (github.com refs only)'
```

Reply:
The reason I chose `PATH` is because it's referring to the `licensee detect [PATH]` in the same doc message. That said, I'm happy to commit one of these, I'd prefer the first.